src/HttpClient/HttpClientImportPogramme.php

```diff
@@ -35,7 +32,7 @@ public function fetchData(): array
     {
         $response = $this->client->request(
             Request::METHOD_GET,
-            $this->httpImportAddress
+            ''
```

Comment:
why empty string? The env var you define should not have the path. Just the domain and maybe a /api or something

Reply:
Ok. Changed value in .env.local